DESIGN.md

```diff
@@ -356,27 +349,15 @@ The final `subscribe` will *not* invoke `cancel`/`dispose` after receiving an `o
 
 ### JVM target and source compatibility
 
-The 2.x version will target JDK6+ but the source will be using some of the Java 8 features.
-We'll use the retrolambda project to generate JDK6 Byte-code from the Java 8 sources.
-The only Java 8 features we intend to use are: lambda expressions, method references and try-with-resources statements.
-Default methods may be considered if retrolambda support reach an acceptable level of maturity.
-
-It still up to discussion to know how we will generate multiple targets?
-Options are:
-
-- two artifcat ids: rxjava-jdk6, rxjava-jdk8
-- classifiers: jdk6, jdk8
-
-The intent is to let Android users consume the new version of RxJava.
-(Not sure if JDK6 is the appropriate requirement, JDK7+ is maybe sufficient).
+The 2.x version will target JDK6+ to let Android users consume the new version of RxJava.
 
-### Future work 
+### Future work
 
-This section contains current design work which needs more discussion and elaboration before it is merged into this document as a stated goal for 2.x. Our goal is 
+This section contains current design work which needs more discussion and elaboration before it is merged into this document as a stated goal for 2.x. Our goal is
 
 #### Custom Observable, Single, Completable, or Flowable
 
-We are investigate a base interface (similar to `Publisher`) for the `Observable`, `Single`, and `Completable` (currently referred to as `Consumable` or `ConsumableObservable`). This would empower library owners and api developers to implement their own type of `Observable`, `Single`, or `Completable` without extending the class. This would result in a change the type signatures of `subscribe` as well as any operator that operates over an `Observable`, `Single`, or `Completable` to accept a more generic type (i.e. `ConsumableObservable`). For more information see the proof of concept project [Consumable](https://github.com/stealthcode/Consumable). 
+We are investigate a base interface (similar to `Publisher`) for the `Observable`, `Single`, and `Completable` (currently referred to as `Consumable` or `ConsumableObservable`). This would empower library owners and api developers to implement their own type of `Observable`, `Single`, or `Completable` without extending the class. This would result in a change the type signatures of `subscribe` as well as any operator that operates over an `Observable`, `Single`, or `Completable` to accept a more generic type (i.e. `ConsumableObservable`). For more information see the proof of concept project [Consumable](https://github.com/stealthcode/Consumable).
```

Comment:
I support the idea of having a separate interface only library. Forcing all semantics into the `Publisher` model is not ideal. 

I have a task to write up more elaboration to the consumable ideas in a separate PR. I suggest that we address this question in that PR. If necessary then delete this section for me to rewrite and propose separately.